workspace/package.json

```diff
@@ -35,8 +35,7 @@
         "@uppy/xhr-upload": "1.7.5",
         "axios": "^0.25.0",
         "bfj": "6.1.1",
-        "codemirror": "^5.52.2",
-        "codemirror-formatting": "1.0.0",
+        "codemirror": "^6.0.1",
```

Comment:
Why do we need `codemirror` lib as depency in Silk? Why is `CodeEditor` from the gui elements not enough?